src/lamplib/src/genny/tasks/mothra_service.py

```diff
@@ -0,0 +1,51 @@
+from dataclasses import dataclass
+import subprocess
+import tempfile
+from typing import Optional
+import structlog
+import yaml
+
+MOTHRA_REPO = "git@github.com:10gen/mothra.git"
+TEAMS_DIR = "mothra/teams"
+TEAMS_FILES = ["cloud.yaml", "database.yaml", "rnd_dev_prod.yaml", "star.yaml"]
+TEAMS_FILES_PATHS = [f"{TEAMS_DIR}/{file}" for file in TEAMS_FILES]
+SLOG = structlog.get_logger(__name__)
+
+
+@dataclass
+class Team:
+    name: str
+    support_slack_channel_name: Optional[str] = None
+    support_slack_channel_id: Optional[str] = None
+
+    def __init__(self, **kwargs):
+        # Filter out any kwargs that are not in the class
+        filtered_kwargs = {k: v for k, v in kwargs.items() if k in self.__annotations__}
```

Comment:
[non-blocking nit]

I'm not a python guy so let me know if this is actually good python. It seems like manually setting these (and letting kwargs absorb the rest) is a better option.

This is actually a perferct use-case for Pydantic, given these are being loaded straight from serialisation, but I'd understand not wanting to pick up that dep in this PR